tests/wrap_stability.rs

```diff
@@ -23,11 +23,12 @@ fn stable_wrap() {
     let mut check_wrap = |text: &_, wrap, start_width| {
         let line = ShapeLine::new(&mut font_system, text, &attrs, Shaping::Advanced);
 
-        let layout_unbounded = line.layout(font_size, start_width, wrap, Some(Align::Left), None);
+        let layout_unbounded =
+            line.layout(font_size, start_width, wrap, Some(Align::Left), None, None);
```

Comment:
nit: might be worth adding stuff like `/* first_line_indent */` to explain the params in the tests, as we start adding more arguments

Reply:
Done.